content/en/getting-started/glossary.md

```diff
@@ -9,4 +9,30 @@ cascade:
 hide_readingtime: true
 ---
 
-Glossary
\ No newline at end of file
+This page defines LocalStack terminologies that you will encounter throughout the documentation.
```

Comment:
This is almost the same sentence as in the "description". IMO it could be removed.